### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by vincent-olivert-riera** [AlfredoRamos/phpbb-ext-imgur#8]
Hello,

I don't know if this is the right place to ask this kind of question, but here it goes. Do you plan to do a new release? I've seen that you made new changes to include a progress bar for the upload process, but such a change is not included in the latest release.

Thanks.

**Comment by AlfredoRamos** [comment]
Hi,

Development on this extension is currently on hold, mostly because I can't test the changes as I would like to.

Everything in the current state (in the master branch) is experimental which means that what you see might or might not be included in the next version, it needs to be tested on real scenarios and the fact that Imgur seems to ban my test accounts with a fair API usage, makes it more difficult for me.

I'll leave this issue open as a reminder.

**Comment by vincent-olivert-riera** [comment]
Hi Alfredo,

first of all, thanks for answering my question.

I'm very interesting on this project, so, please could you tell me how can I help you to test it?

Thanks.

**Comment by AlfredoRamos** [comment]
Hi,

You could help me testing development versions, if you don't use GNU/Linux I could provide a packaged file ready to unzip and install, otherwise you would need Composer, to install and generate the package, see the README.md for instructions.

I try to push changes to the master branch only after I tested them to prevent critical or security bugs, but nevertheless I wouldn't recommend to use a development version on live board unless is a test one.

[alfredoramos_imgur_1.2.0-dev_20181012.zip](https://www.dropbox.com/s/f5f06fpuvv2ysq9/alfredoramos_imgur_1.2.0-dev_20181012.zip?dl=1)

I would only need feedback on errors you encounter or things that you don't like.
